index.js

```diff
@@ -12,6 +12,7 @@ const dashRegex        = require('dash-regex');
 const moneroRegex      = require('monero-regex');
 const rippleRegex      = require('ripple-regex');
 const neoRegex         = require('neo-regex');
+const TronWeb          = require('tronweb');
```

Comment:
This is too heavy a library to import here just for use of `TronWeb.utils.crypto.isAddressValid()`. It would make more sense if we could only import that specific method or create a new NPM package that simply corresponds to this.